src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -882,7 +886,11 @@ public class CaseData extends Et1CaseData {
     @JsonProperty("referralDetails")
     private String referralDetails;
     @JsonProperty("referralDocument")
-    private String referralDocument;
+    private List<DocumentTypeItem> referralDocument;
```

Comment:
again just a heads up on changing types, can cause issues but given this isn't live yet, it's 'ok'